### DISCUSSION TITLE
Replacing reset layouts with named layouts

### DISCUSSION THREAD

**Discussion by BillBuilt** [sveltejs/kit#4558] [General]
I decided it was time to do a clean update to a sveltekit project i've been working on. Upon running `npm install` after deleting `/node_modules` and `package-lock.json`, got bit by the following error  `__layout.reset has been removed in favour of named layouts:`. Just last night I watched Rich's talk on the new fetures in sveltekit, but I guess I didn't realize this was a breaking change. So in case anyone finds themselve in the same pickle, here is what I came up with, and so far it's working well.

So my old layout scheme was something like this:
```
src/routes/
├ design/
│ ├ editor/
│ │ ├ [action]-[id_or_type].svelte
│ │ ├ [action]-[id_or_type].ts
│ │ └ __layout.reset.svelte
├ index.svelte
└ __layout.svelte
```

Simply renaming `[action]-[id_or_type].*` to `[action]-[id_or_type]@editor.*` and `__layout.reset.svelte` to `__layout-editor.svelte` caused `__layout-editor.svelte` to be embedded (nested) into `__layout.svelte`

The workaround I came up with was:

```
src/routes/
├ design/
│ ├ editor/
│ │ ├ [action]-[id_or_type].svelte
│ │ ├ [action]-[id_or_type].ts
│ │ └ __layout@reset.svelte
├ index.svelte
├ __layout.svelte
└ __layout-reset.svelte
```
 
Where `__layout-reset.svelte` simply contains a single empty slot `<slot />` This seems to reproduce the old behavior, although I admit I have only just started down this path but hope this helps someone get started down theirs.

**Comment by GRASBOCK** [comment]
Here i some additional info on that in [SO](https://stackoverflow.com/a/71789963)
